config/CLI/halfunet.yaml

```diff
@@ -0,0 +1,13 @@
+model:
+  model_name: "halfunet"
+  model_conf: null
+  num_inter_steps: 1
+  lr: 1e-4
+  loss: "mse"
+  training_strategy: "diff_ar"
+  use_lr_scheduler: False
+  precision: "32"
+
+  no_log: False # doublon avec trainer.logger
```

Comment:
i think we can remove the `no_log` argument and replace it with `if self.trainer.logger is None`

Reply:
It says that the AutoRegressiveLightning is not attached to a `Trainer`. It is weird because it access to the trainer later, for instance in the configure_optimizers method.